include/aos/common/config.hpp

```diff
@@ -359,6 +359,13 @@
 #define AOS_CONFIG_PKCS11_USE_STATIC_LIB
 #endif
 
+/**
+ * Default PKCS11 library.
+ */
+#ifndef AOS_CONFIG_PKCS11_DEFAULT_LIB
+#define AOS_CONFIG_PKCS11_DEFAULT_LIB "/usr/lib/softhsm/libsofthsm2.so"
```

Comment:
It should be CRYPTOUTILS param. As well as it should be handled in cryptutils.

Reply:
fixed